api/models/GroupRelationshipInvite.js

```diff
@@ -147,12 +147,14 @@ module.exports = bookshelf.Model.extend(Object.assign({
         const parentToChild = invite.get('type') === GroupRelationshipInvite.TYPE.ParentToChild
         const childToParent = invite.get('type') === GroupRelationshipInvite.TYPE.ChildToParent
 
-        // If child group is hidden then only tell moderators of the parent about it joining the parent, otherwise tell all the parent's members about it
-        const fromMembers = (childToParent && fromGroup.isHidden()) ? await fromGroup.moderators().fetch({ transacting }) : await fromGroup.members().fetch({ transacting })
-        const toMembers = (parentToChild && toGroup.isHidden()) ? await toGroup.moderators().fetch({ transacting }) : await toGroup.members().fetch({ transacting })
+        // If child group is hidden then only tell administrators of the parent about it joining the parent, otherwise tell all the parent's members about it
+        // TODO: RESP. check if this is what we want
+        const fromMembers = (childToParent && fromGroup.isHidden()) ? await fromGroup.stewards().fetch({ transacting }) : await fromGroup.members().fetch({ transacting })
```

Comment:
on a side note; I got a push notif about this event where the names of the groups were undefined, undefined. But the notification in app was working fine, so something is wrong specifically with the push notif